exercises/sum.cpp

```diff
@@ -6,3 +6,15 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+using namespace std;
+int main(){
+	int a, b, sum;
+	cout << "insert the first number:";
+	cin >> a;
+	cout << "insert the second number:";
+	cin >> b;
+	sum = a+b;
+	cout << "Sum:" << sum;
```

Comment:
manca il return
inoltre potresti mettere qualche riga in più per migliorare la leggibilità del codice

la variabile sum potresti non usarla, quindi cancellarlo